### ISSUE TITLE
False positive `RequireAbstractOrFinal.ClassNeitherAbstractNorFinal` for `final readonly class`

### ISSUE THREAD

**Issue by simPod** [slevomat/coding-standard#1485]
```php
final readonly class Foo
{
}
```

gives `All classes should be declared using either the "abstract" or "final" keyword.`

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/9b0fc3084ed147bbd5a66df8dd6e86cdc03c3f2c
